projects/core/directives/dropdown/dropdown-hover.directive.ts

```diff
@@ -12,10 +12,16 @@ export class TuiDropdownHoverDirective extends TuiDriver {
     private readonly toggle$ = new Subject<boolean>();
     private readonly stream$ = merge(this.toggle$, this.hovered$).pipe(
         switchMap(visible =>
-            of(visible).pipe(delay(visible ? this.showDelay : this.hideDelay)),
+            of(visible).pipe(
+                delay(visible ? this.showDelay : this.hideDelay),
+                tap(visible => this.openChange.emit(visible)),
```

Comment:
this breaks hoverable/clickable case: once clicked, the dropdown shouldn't close until click outside dropdown

https://user-images.githubusercontent.com/46284632/212109236-7063ede3-7956-42b1-9e80-7920cb5f69a2.mov

Reply:
fixed